tests/unit/amo/components/TestSearchContextCard.js

```diff
@@ -0,0 +1,158 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import {
+  SearchContextCardBase,
+  mapStateToProps,
+} from 'amo/components/SearchContextCard';
+import { searchLoad, searchStart } from 'core/actions/search';
+import { dispatchClientMetadata, fakeAddon } from 'tests/unit/amo/helpers';
+import { getFakeI18nInst } from 'tests/unit/helpers';
+
+
+describe('SearchContextCard', () => {
+  function render(props = {}) {
+    let { store } = props;
+
+    if (!store) {
+      store = dispatchClientMetadata().store;
+      store.dispatch(searchStart({ filters: { query: 'test' } }));
+    }
+
+    return shallow(
+      <SearchContextCardBase
+        {...mapStateToProps(store.getState())}
```

Comment:
I was with you until `return root.first().shallow(options).first().shallow(options);` 😆

I suppose we could write a helper for that, although I liked importing things directly from `enzyme`, it felt so clean!

What happens when we have multiple things in the `compose` chain like a `connect()` call and a `translate()` call?